### PR TITLE
Astro Release Notes (June 9th)

### PR DESCRIPTION
**Pull request by jwitz**
Based on https://app.zenhub.com/workspaces/developer-productivity-and-readiness-6242176c89ed9000197ec734/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvNzc1MjA

Tagging @chomipi88 in case I missed any cloud-level changes this week.

### CODE REVIEW

File: astro/api-keys.md [hunk 1]
```diff
@@ -45,10 +49,24 @@ If you just need to make a single API call, you can use a temporary user authent
...
+- Deploy code to Astro [using CI/CD](ci-cd.md) with tools such as GitHub Actions or Jenkins.
+- Deploy code and configuration changes to Astro [using the Astro CLI](deploy-code.md) without user authentication.
 - Automate requests to the [Airflow REST API](airflow-api.md).
 
+To use API keys with the Astro CLI, you need to make your Deployment API key ID and secret accessible to the machine running the CLI via the following OS-level environment variables:
+
+- `ASTRONOMER_KEY_ID`
+- `ASTRONOMER_KEY_SECRET`
+
+For example, to update a given Deployment using the Astro CLI on a Mac machine, you can set temporary OS-level environment variables using the following command:
```
Comment (jwitz) [inline astro/api-keys.md:61]:
```suggestion
For example, to update a given Deployment using the Astro CLI on a Mac machine, set temporary OS-level environment variables with the following commands:
```

File: astro/api-keys.md [hunk 2]
```diff
@@ -45,10 +49,24 @@ If you just need to make a single API call, you can use a temporary user authent
...
+- `ASTRONOMER_KEY_SECRET`
+
+For example, to update a given Deployment using the Astro CLI on a Mac machine, set temporary OS-level environment variables with the following commands:
+
+```sh
+export ASTRONOMER_KEY_ID=<your-key-id>
+export ASTRONOMER_KEY_SECRET=<your-key-id>
+```
+
+After setting the variables, running `astro deployment update` works for the Deployment without you having to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```
Comment (kwcanuck) [inline astro/api-keys.md:68]:
```suggestion
After setting the variables, running `astro deployment update` works for the Deployment and you don't need to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```

File: astro/cli/astro-deployment-list.md [hunk 3]
```diff
@@ -15,6 +15,17 @@ List all Deployments within your current Workspace.
...
 ```
 
+:::tip
+
+To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md), set the following OS-level environment variables in a way that the Astro CLI can access them:
+
+- `ASTRONOMER_KEY_ID`
+- `ASTRONOMER_KEY_SECRET`
+
+After setting the variables, this command works for a Deployment without you having to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```
Comment (kwcanuck) [inline astro/cli/astro-deployment-list.md:25]:
```suggestion
After setting the variables, this command works for a Deployment and you don't need to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```

**Comment by chomipi88** [comment]
@jwitz nothing on the cloud or infra side to call out this week

File: astro/cli/astro-deployment-list.md [hunk 4]
```diff
@@ -15,6 +15,17 @@ List all Deployments within your current Workspace.
 astro deployment list
 ```
 
+:::tip
+
+To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md), set the following OS-level environment variables in a way that the Astro CLI can access them:
```
Comment (kwcanuck) [inline astro/cli/astro-deployment-list.md:20]:
```suggestion
To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md) and make the variables accessible to the Astro CLI, set the following OS-level environment variables:
```

**Review by chomipi88 — APPROVED** [review]

File: astro/cli/astro-deployment-update.md [hunk 5]
```diff
@@ -15,6 +15,17 @@ Update the configuration for a Deployment on Astro. This command is functionally
 astro deployment update <deployment-id> <flags>
 ```
 
+:::tip
+
+To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md), set the following OS-level environment variables in a way that the Astro CLI can access them:
```
Comment (kwcanuck) [inline astro/cli/astro-deployment-update.md:20]:
```suggestion
To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md) and make the variables accessible to the Astro CLI, set the following OS-level environment variables:
```

File: astro/cli/astro-deployment-update.md [hunk 6]
```diff
@@ -15,6 +15,17 @@ Update the configuration for a Deployment on Astro. This command is functionally
...
+
+- `ASTRONOMER_KEY_ID`
+- `ASTRONOMER_KEY_SECRET`
+
+After setting the variables, this command works for a Deployment without you having to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```
Comment (kwcanuck) [inline astro/cli/astro-deployment-update.md:25]:
```suggestion
After setting the variables, this command works for a Deployment you don't need to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```

File: astro/cli/astro-deployment-variable-create.md [hunk 7]
```diff
@@ -11,18 +11,23 @@ For a given Deployment on Astro, create environment variables via the Astro CLI
...
-
 ## Usage
 
 ```sh
 astro deployment variable create
 ```
 
+:::tip
+
+To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md), set the following OS-level environment variables in a way that the Astro CLI can access them:
```
Comment (kwcanuck) [inline astro/cli/astro-deployment-variable-create.md:22]:
```suggestion
To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md) and make the variables accessible to the Astro CLI, set the following OS-level environment variables:
```

File: astro/cli/astro-deployment-variable-list.md [hunk 8]
```diff
@@ -19,6 +19,17 @@ If an environment variable value is set as secret, the CLI will list only its ke
...
 ```
 
+:::tip
+
+To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md), set the following OS-level environment variables in a way that the Astro CLI can access them:
+
+- `ASTRONOMER_KEY_ID`
+- `ASTRONOMER_KEY_SECRET`
+
+After setting the variables, this command works for a Deployment without you having to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```
Comment (kwcanuck) [inline astro/cli/astro-deployment-variable-list.md:29]:
```suggestion
After setting the variables, this command works for a Deployment and you don't need to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```

File: astro/cli/astro-deployment-variable-update.md [hunk 9]
```diff
@@ -22,6 +22,17 @@ This command is functionally identical to editing and saving the `value` of an e
 astro deployment variable update
 ```
 
+:::tip
+
+To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md), set the following OS-level environment variables in a way that the Astro CLI can access them:
```
Comment (kwcanuck) [inline astro/cli/astro-deployment-variable-update.md:27]:
```suggestion
To run this command in an automated process such as a [CI/CD pipeline](ci-cd.md) and make the variables accessible to the Astro CLI, set the following OS-level environment variables:
```

File: astro/cli/astro-deployment-variable-update.md [hunk 10]
```diff
@@ -22,6 +22,17 @@ This command is functionally identical to editing and saving the `value` of an e
...
+
+- `ASTRONOMER_KEY_ID`
+- `ASTRONOMER_KEY_SECRET`
+
+After setting the variables, this command works for a Deployment without you having to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```
Comment (kwcanuck) [inline astro/cli/astro-deployment-variable-update.md:32]:
```suggestion
After setting the variables, this command works for a Deployment and you don't need to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```

File: astro/environment-variables.md [hunk 11]
```diff
@@ -49,6 +49,8 @@ To avoid exposing secret values in task logs, instruct your team to not log envi
 
 :::
 
+If you prefer to work with the Astro CLI, you can also create and update environment variables using the `astro deployment variable create` and `astro deployment variable update` commands. For more information, see [CLI Command Reference](cli/astro-deployment-variable-create.md).
```
Comment (kwcanuck) [inline astro/environment-variables.md:52]:
```suggestion
If you prefer to work with the Astro CLI, you can create and update environment variables using the `astro deployment variable create` and `astro deployment variable update` commands. For more information, see [CLI Command Reference](cli/astro-deployment-variable-create.md).
```

**Review by kwcanuck — APPROVED** [review]
Everything looks good. I've suggested minor changes. There is no need to implement them if you disagree.

File: astro/cli/astro-deployment-list.md (hunk 4 above)
Comment (jwitz) [reply astro/cli/astro-deployment-list.md:20]: Users aren't setting out to "Make the variables accessible". That is a step necessary for using an automated process, which is the main end goal here. These variables have no meaning to users beyond that they're setup steps.

File: astro/cli/astro-deployment-update.md (hunk 5 above)
Comment (jwitz) [reply astro/cli/astro-deployment-update.md:20]: See previous comment

**Pull request merged**